docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,833 @@
+{
```

Comment:
<p>I think here the explanation could be a bit more extensive as well. I think it's good idea to mention the advantages of structural models but maybe we could relate it a bit more to the plots? </p><p> </p><p>The way it is formulated here I think it sounds a bit like increasing the return to fishing is like implementing a tuition subsidy which isn't the case I think (correct me if I am wrong). Also, if possible, it might be nice to select a value for <code>exp_fishing</code> that doesn't eliminate one whole choice or (even better) show the results for a range of values to demonstrate how varying just one parameter can lead to very different outcomes.  I think the plots are quite self-explanatory, but since this is basically the 101 tutorial, it might be good to be very explicit and describe the plots in detail. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Hi Annica,
Thank you for your comments!
1) Yes, some rephrasing is definitely needed. I actually was thinking to extend that section and insert a cross-reference to the explanations/how-to guide, if we include some general discussion on structural models in general.  
2) I was lazy there :D I like the idea of a range of values, the description would also be more interesting for sure.